### DISCUSSION TITLE
j not down, k now up, but h, l, and all arrow work well

### DISCUSSION THREAD

**Discussion by laowalter** [jarun/nnn#1706] [General]
Hi everyone,

I'm having a problem with nnn in my i3wm environment. The j and k keys are not working as expected. They should be used to navigate up and down in the terminal, but they are not doing anything, mean while, the key h works fine.

 I have tried a few different things to fix this, but nothing has worked so far. The j and k keys work fine in ranger, so I don't think the i3wm is hijacking them.

Any help would be greatly appreciated.

Thanks,

**Comment by jarun** [comment]
This looks to be an environment/WM issue rather than an `nnn` issue. Please debug yourself or contact i3wm maintainers.
